elcid/urls.py

```diff
@@ -18,6 +18,11 @@
     url(r'stories/$', views.TemplateView.as_view(template_name='stories.html')),
     url(r'elcid/v0.1/', include(api.elcid_router.urls)),
     url(r'labtest/v0.1/', include(api.lab_test_router.urls)),
+    url(
+        r'^elcid/renal_handover',
```

Comment:
We're prefixing urls with the name of the application now ? ;)

Reply:
As discussed this is to namespace the views in terms of the service that's using them.
Definitely true that using elcid as a sometime synonym for infection_service is not idea.